### ISSUE TITLE
Exception thrown when adding a snake without a color customization

### ISSUE THREAD

**Issue by xtagon** [smallsco/mojave#6]
Hi,

I encountered this error when trying to add a snake:

```
Error

modules/Utils.lua:259: attempt to index local 'value' (a nil value)


Traceback

modules/Utils.lua:259: in function 'color_from_hex'
modules/menu_panes/SnakesPane.lua:162: in function 'draw'
modules/Menu.lua:149: in function 'draw'
main.lua:134: in function 'draw'
[C]: in function 'xpcall'
```

Now, when I restart Mojave, if I click on Manage Snakes again the same error occurs.

Mojave Version: 3.1

**Comment by smallsco** [comment]
What kind of a snake are you adding? API v1 or something else?  What value is your snake returning for "color" in the root endpoint (or start endpoint for older APIs)? I suspect that I'm forgetting to check for null and should probably just assign a random color if that's the case.

To get your Manage Snakes pane back, go to https://github.com/smallsco/mojave#troubleshooting and grab the data directory path for your OS, inside the `mojave3` folder you will find a `snakes.json` file. You can delete that file and restart the app (this will remove all your snakes) or you can manually edit the file to add a color value to the snake that you added.

**Comment by xtagon** [comment]
It's API v1, but you're right, my snake is not returning a color or any other customizations. The index route returns:

```
{"apiversion":"1","author":"xtagon"}
```

Deleting `snakes.json` fixed Manage Snakes as you said.

Thanks for your help! Knowing it's the color, I can work around this.

**Comment by smallsco** [comment]
I've just published version 3.1.1 which should fix this issue. Can you confirm that it works correctly for you using the latest release?

**Comment by xtagon** [comment]
Works great in 3.1.1. Thanks again!
